### DISCUSSION TITLE
Delete wire in circuitbox deletes label

### DISCUSSION THREAD

**Discussion by FredrikM97** [FakeFishGames/Barotrauma#14072] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

When I edit a circuit box and delete a wire that have a label behind it the label is also removed even though it is not selected. Wire seems to have the options Add label, edit label and delete. Not possible to only select the wire. Label is also automatically selected.
### Reproduction steps

1. Login to multiplayer server
2. Enter circuit box
3. Create a label on top a wire
5. right click a wire
6. Click delete

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in our private repo, fix coming up in a future update!
